Inventory.cs

```diff
@@ -0,0 +1,82 @@
+﻿public class Inventory
+{
+    private List<Product> products = new();
+
+    public void Add(string name, decimal price, int quantity)
```

Comment:
rename to **AddProduct** 
and u can recieve a product object instead of 3 params